samples/nrf9160/aws_fota/README.rst

```diff
@@ -29,6 +31,57 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+Prerequisites
+*************
+
+The connected device must have the same MQTT Client ID as the thing name created on AWS IoT.
+It also needs a policy attached to the certificates, and the certificates need to be active.
+
+Below is a very permissive policy which should be used for **testing only** which you can attach to the certificates if you are unsure on what policies are needed for your device. To create a policy go to `AWS IoT Console
+<https://console.aws.amazon.com/iot/home>`_ Secure>Policies>Create.
+
+.. code-block:: javascript
+
+    {
+      "Version": "2012-10-17",
+      "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": "iot:*",
+            "Resource": "*"
+          }
+       ]
+     }
+
+To create a thing log in to the `AWS IoT Console
+<https://console.aws.amazon.com/iot/home>`_ Manage>Things>Create>Create a single thing.
+Give it a name which corresponds to your MQTT Client ID.
+Chose next either add your certificates or chose to create certificates, press activates and remember to download one of the `root CA certificates
+<https://docs.aws.amazon.com/iot/latest/developerguide/managing-device-certs.html#server-authentication>`_ and the generated certificates.
+
+Afterward, attach the policy you created before and put the information from the Root CA, Public Certificate, and Private Key into `certificates.h`.
+After this is done deselect the option in menuconfig which is described as *Use provisioned certificates*.
+This will write your new certificates to the secure tag defined in menuconfig into the modem.
```

Comment:
Can you link to somewhere where secure tags are described?

Reply:
Would a link to https://docs.zephyrproject.org/latest/reference/networking/sockets.html?highlight=sec_tag#_CPPv39sec_tag_t suffice?

Reply:
That's great. What does "defined in menuconfig" mean?

Reply:
added an :option: tag but it's an option in menuconfig where you can define the number/"tag" which the certificates are stored at.(It's the key to get the certificates) so if you give the key into the sockets they will use the certificates stored at that key value.